File: DOTNET/C#/ConsoleApplications/valuetype/example1.cs
using System;

struct mystruct
{
	public int age;
	public string name;
	public string Name
	{
		get{return name;}
		set{name = value;}
	}
	public int Age
	{
		get{return age;}
		set{age = value;}
	}
}
class exe
{
	public static void Main()
	{
		mystruct my = new mystruct();
		my.name = "arif";
		my.age = 22;
		Console.WriteLine(my.Name + " " + my.Age);
		//Console.WriteLine(m.Name + " " + m.Age);
	
	}
}

File: CPlusPlus/constructor/example1.cpp
#include <iostream>

using namespace std;

struct myclass
{
	public:
	    myclass(); //default constructor
	    myclass(int age, char* fname, char* lname, char* mname);
	    
	    private:
	    int myage;
	    char* fname;
	    char* lname;
	    char* mname;
	    public:
	       char* getfname();
	       char* getlname();
	       char* getmname();
	       int getage();
};
myclass::myclass()
{
	cout<<"default constructor called \n";
}
	    myclass::myclass(int age, char* fname, char* lname, char* mname)
	    {
	    	myage = age;
	    	fname = fname;
	    	lname = lname;
	    	mname = mname;
}
char* myclass::getfname()
{
	return fname;
}
char* myclass::getlname()
{
	return lname;
}
char* myclass::getmname()
{
	return mname;
}
int myclass::getage()
{
	return myage;
}
int main()
{
	myclass cl(22, "arif", "khan", "hasan");
	cout<<"your first name is "<<endl;
	char* name = cl.getfname();
	cout<<name;
	return 0;
}